### PR TITLE
add ecommerce blog post part 1

### PR DESCRIPTION
**Pull request by vkarpov15**
## What was changed:

First blog post :+1:

## Checklist

1. Closes issue: 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: blog/2021-05-12-build-an-ecommerce-app-with-temporal-part-1.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+	return nil
+}
+```
+
+To run a Workflow, you need to create a Worker process.
+A Temporal _Worker_ listens for events on a queue and has a list of registered Workflows that it can run in response to messages on the queue.
+Below is the largely-boilerplate `worker/main.go` file:
+
+<!--SNIPSTART temporal-ecommerce-worker-->
```
Comment (flossypurse) [inline blog/2021-05-12-build-an-ecommerce-app-with-temporal-part-1.md:94]: If you have these placeholders here, then no need to check in the code between the placeholders.

The snippet gets merged in at build time.

**Review by flossypurse — APPROVED** [review]
Wicked!
Let's do it!

**Comment by sw-yx** [comment]
from what i understand, @mfateev had some comments for edits to the code here, @rylandg will confirm but we intend to ship this post tomorrow

**Review by rylandg — APPROVED** [review]
LGTM

**Pull request merged**